docker-compose/readme.md

```diff
@@ -20,6 +20,15 @@ To start a full Grid backend stack one needs the following:
 - min of 32GB RAM
 - min of 1TB SSD storage (high preference for NVMe based storage)
 
+#### Setting the DNS Records
+
+The following table explicitly shows how to set the A and AAAA records for your domain.
+
+| Type | Host | Value          |
+| ---- | ---- | -------------- |
+| A    | \*   | <ipv4_address> |
```

Comment:
I'm not sure you need the \ in front of the *. We use name.com and we only need to add *

Reply:
Yes indeed it's because the file is in markdown. It's to be sure that we see the symbol.